src/LSST/M1M3/SS/FPGA/SimulatedFPGA.cpp

```diff
@@ -49,6 +51,38 @@ SimulatedFPGA::SimulatedFPGA() {
         this->rnd[i] = float((rand() % 2000) - 1000) / 1000.0;
     }
     this->rndIndex = 0;
+    _mgrMTMount = SAL_MTMount();
+    _mgrMTMount.salTelemetrySub(const_cast<char*>("MTMount_Elevation"));
+
+    _monitorMountElevationThread = std::thread(&SimulatedFPGA::_monitorElevation, this);
+}
+
+SimulatedFPGA::~SimulatedFPGA() {
+    _exitThread = true;
+
+    _monitorMountElevationThread.join();
+
+    _mgrMTMount.salShutdown();
+}
+
+void SimulatedFPGA::_monitorElevation(void) {
+    MTMount_ElevationC mountElevationInstance;
+
+    spdlog::debug("Start monitoring mount elevation...");
+
+    while (!_exitThread) {
+        ReturnCode_t status = _mgrMTMount.getSample_Elevation(&mountElevationInstance);
+
+        if (status == 0) {
+            spdlog::debug("Got valid elevation sample...");
+
+            {
+                std::lock_guard<std::mutex> lock_g(_elevationReadWriteLock);
+                _mountElevation = mountElevationInstance.Elevation_Angle_Actual;
+            }
+        }
+        std::this_thread::sleep_for(std::chrono::milliseconds(20000));
```

Comment:
It was originally usleep(20000). Should now be std::chrono::milliseconds(20) - usleep sleeps for microseconds, which is ms * 1000